tests/test_report/test_report.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
What are tests/report/test_report and tests/test_report/test_report meant to be? Also why are they  in different folders?

Reply:
This shouldn't be here. I'll remove it.